src/fides/api/ctl/sql_models.py

```diff
@@ -342,7 +348,7 @@ def get_system_data_uses(
         Utility method to get any data use that is associated with at least one System
         """
         data_uses = set()
-        for row in db.query(System.privacy_declarations).all():
```

Comment:
Optional, but I wrote the`System.collapse_data_uses` in its current form so you could continue to just select the System.privacy_declarations field here instead of all the fields and still share some code with `get_system_data_uses`.  

Now that we have to fetch the whole system anyway, we could just consolidate the three methods: `collapse_data_uses`, `get_system_data_uses`, and `get_data_uses`  into one single class method that takes in a list of systems, and an `include_parents` bool. `get_system_data_uses` now seems unnecessarily complicated.

Reply:
updated in https://github.com/ethyca/fides/pull/3098/commits/7711f84099fb6a5f623e780c466adadb22ef1597

Reply:
thank you Adam